### PR TITLE
Small optimisation identify_client

### PR DESCRIPTION
**Pull request by pavel-pimenov**

### CODE REVIEW

File: src/identify_client.cpp [hunk 1]
```diff
@@ -273,7 +274,6 @@ namespace
 		, {0, "-G3", "G3 Torrent"}
 		, {0, "-FG", "FlashGet"}
 		, {0, "-ML", "MLdonkey"}
-		, {0, "-MG", "Media Get"}
```
Comment (arvidn) [inline src/identify_client.cpp]: why did this move?

Comment (pavel-pimenov) [inline src/identify_client.cpp]: I counted the number of hits
Revert... "Media Get" only russian locations

**Comment by arvidn** [comment]
did you see this? https://travis-ci.org/arvidn/libtorrent/jobs/179440439#L1295

it's an unsigned  <-> signed comparison

File: src/identify_client.cpp [hunk 2]
```diff
@@ -137,7 +137,7 @@ namespace
 
 	// only support BitTorrentSpecification
 	// must be ordered alphabetically
-	map_entry name_map[] =
+	static map_entry name_map[] =
```
Comment (arvidn) [inline src/identify_client.cpp]: these should probably be ``const`` too

File: src/identify_client.cpp [hunk 3]
```diff
@@ -367,15 +367,14 @@ namespace libtorrent
...
 		if (p.is_all_zeros()) return "Unknown";
 
 		// ----------------------
 		// non standard encodings
 		// ----------------------
 
-		int num_generic_mappings = sizeof(generic_mappings) / sizeof(generic_mappings[0]);
+		const int num_generic_mappings = sizeof(generic_mappings) / sizeof(generic_mappings[0]);
 
 		for (int i = 0; i < num_generic_mappings; ++i)
```
Comment (arvidn) [inline src/identify_client.cpp]: to luxury it up a bit, this could even be a for-range loop over the C-array

**Pull request merged**